src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/Event.kt

```diff
@@ -46,8 +53,10 @@ data class Event constructor(
   @Column(name = "eventable_id")
   val eventableId: String,
 
-  @Transient
-  val details: Map<String, Any>?,
+  @JsonDetailsConverter
+  @Convert(converter = JpaDetailsConverter::class)
```

Comment:
This is JPA wizardry. Essentially this gets flattened out into a JSON string representation of the details from the original event json e.g. `{"vehicle_reg": "ABCDEDF"}`